x-pack/plugins/screenshotting/server/screenshots/open_url.ts

```diff
@@ -29,6 +29,21 @@ export const openUrl = async (
 
   try {
     await browser.open(url, { context, headers, waitForSelector, timeout }, kbnLogger);
+
+    // debugging statements to trace puppeteer behavior
+    await browser.evaluate(
+      {
+        fn: () => {
+          window.addEventListener('resize', () => {
+            // eslint-disable-next-line no-console
+            console.log(`window resize, width is now: ${window.innerWidth}px`);
```

Comment:
This logging will only be seen for print layout PDF report jobs, currently.

This logging would be seen during a screenshot capture, if not for the fix in chromium/driver.ts. However, it does track resize events that happen in the page setup for print layout PDF report jobs.